### PR TITLE
Adding a directories list

### PR DESCRIPTION
**Pull request by RichardLitt**
This is a basic directories list. For the README, it should give an entry into the org more than the basic list used currently on the @orbitdb page.

### CODE REVIEW

File: directories-list.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+- [orbit-db-store](https://github.com/orbitdb/orbit-db-store)
+
+This is a base calls for all datastores in orbitdb. A Base Class defines the common interface for all databases. We should refactor this - it should not be a base class, but a composting (?). We'll figure this out. The Stores are the API of that database; they also contian the index so that if you cac he anything in the memory, the user never actually directly asks the index, but the API of the store. All of these stores use -- they're very simple, there's no io or so on. The store is where that happens. Every store calls this operation which calls the log, and so on, and eventually sends a cb somewhere saying that it is persisted. It has a load method that checks the cache, then ... I lost it. The base class hooks the orbitdb main thing, that contains the messaging and the keystore and the access-controller with the log and the form and so on with the indices.
```
Comment (rjharmon) [inline directories-list.md]: s/calls/class
s/contian/contain
s/cac he/cache

Comment (RichardLitt) [reply directories-list.md]: I think this was for an older commit? :)

**Comment by rjharmon** [comment]
whoops, sorry :)

**Comment by RichardLitt** [comment]
This is better than what we have. Errors or not, I am going to merge it now. If anyone has suggestions, feel free to open a new PR for them!

**Pull request merged**